src/Personnummer.php

```diff
@@ -347,6 +349,10 @@ public function getAge(): int
             $day -= 60;
         }
 
+        if ($this->isSllReserveNumber()) {
```

Comment:
Return a negative value since century and year does not represent the year of birth.